### ISSUE TITLE
Expand ctrl/shift search behaviour to search different parts of commands

### ISSUE THREAD

**Issue by ZetaDaemon** [BLCM/OpenBLCMM#47]
command general syntax is `set object property value`
holding control or shift while searching will search the entire command, would be nice to be able to search for either in the object or in the value rather than the whole command, with options to set the behaviour.
as an example setup
ctrl: search object
shift: search value
ctrl shift: search whole command

**Comment by apocalyptech** [comment]
Yep, I dig it.  Extra settings for the "Input" menu; would maybe default all three to "search whole command" just so I'm not yanking functionality out from anyone, but would be groovy.
